python/setup.py

```diff
@@ -4,17 +4,13 @@
 
 import subprocess
 
-from setuptools import setup, find_packages
-import setuptools.command.install as _install
+from setuptools import setup, find_packages, Extension, Distribution
+import setuptools.command.build_ext as _build_ext
 
 
-class install(_install.install):
+class build_ext(_build_ext.build_ext):
   def run(self):
     subprocess.check_call(["../build.sh"])
```

Comment:
I see, the extension modules get copied automatically if you set `self.extensions` or something like that in the `build_ext` class? I'll see if I can get something like that to work. Thanks!